atomic_reactor/plugins/post_generate_maven_metadata.py

```diff
@@ -56,6 +66,101 @@ def get_nvr_components(self, nvr_requests):
 
         return components
 
+    def download_sources(self, url_requests):
```

Comment:
Cyclomatic complexity of this function is overwhelming to me. I got lost after 50 lines of reading. Can we abstract some parts?

Reply:
Split it into two functions to reduce complexity. Both functions are now <=50 lines.